jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet

```diff
@@ -89,7 +89,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--path.procfs=/host/proc',
           '--path.sysfs=/host/sys',
           '--path.rootfs=/host/root',
-
+          '--no-collector.wifi',
+          '--no-collector.hwmon',
+          '--no-collector.netstat',
```

Comment:
This one and conntrack needs some thought as those might affect debuggability of networking issues.